src/component/menu/drawer.jsx

```diff
@@ -99,6 +100,10 @@ export const DrawerMenu = ({
                         {item.title}
                     </NavLink>
                 ))}
+                <a className={classnames(styles.navigationLink)} href={`${getBasePath()}/logout`}>
+                    {getIcon('exit_to_app')}
```

Comment:
Just merged a PR on the offline capabilities. With these changes, this needs to be changed to: 

```
import IconName from 'material-ui/core'

getIcon(IconName)
```